src/auro-themeswitcher.js

```diff
@@ -0,0 +1,236 @@
+// Copyright (c) Alaska Air. All right reserved. Licensed under the Apache-2.0 license
+// See LICENSE in the project root for license information.
+
+// ---------------------------------------------------------------------
+
+import { LitElement, html } from "lit";
+import styleCss from "./theme-switcher-style-css.js";
+
+/**
+ * @attr {Array} themes - This accepts an array of JSON object outlining the themes to support.
+ */
+
+/* eslint-disable max-statements, one-var, no-magic-numbers */
+export class AuroThemeSwitcher extends LitElement {
+
+  constructor() {
+    super();
+    this.themes = [
+      {
+        label: 'Auro',
+        url: 'https://cdn.jsdelivr.net/npm/@alaskaairux/design-tokens@latest/dist/tokens/CSSCustomProperties.css'
+      },
+      {
+        label: 'Auro Dark',
+        url: 'AURO DARK URL'
```

Comment:
Going to darkmode is not a process of switching a URL path to tokens. The darkmode token definitions should be loaded with the light mode defs, only that the darkmode is wrapped in this media query. 

```
@media screen and (prefers-color-scheme: dark) {
    :root {
```

We also need this to trigger the rest of the browser's native color modes. 

```
@media screen and (prefers-color-scheme: dark) {
    :root {
        color-scheme:dark
    }
}
```

Now if I understand this correctly, the button function will not swap out URls, but update we can update a style tag in the header of the page to toggle the following and that would work the same as the OS preferences. 

```
<html lang="en" style="color-scheme: light/dark;">
```